dsp-user/core/src/test/scala/dsp/user/domain/UserDomainSpec.scala

```diff
@@ -17,7 +17,7 @@ import dsp.valueobjects.User._
  */
 object UserDomainSpec extends ZIOSpecDefault {
 
-  def spec = (compareUsersTest + createUserTest + updateUserTest)
+  def spec = suite("UserDomainSpec")(compareUsersTest, createUserTest, updateUserTest)
```

Comment:
any particular reason for this change? should this be done everywhere?

Reply:
According to Ivan it is a recommendation by John to do it like that. The pattern is: the "root suite" should be called the same as the name of the file that is checked. If you like it, we can introduce it everywhere.

Reply:
We can discuss it in our API meeting. I add it to the list of topics.